sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixTestKit.java

```diff
@@ -156,18 +160,33 @@ public Settings withAdvancedViews() {
      * @return The updated settings.
      */
     public Settings withWorkflowTickInterval(Duration tickInterval) {
-      return new Settings(stopTimeout, serviceName, aclEnabled, true, Optional.of(tickInterval));
+      return new Settings(stopTimeout, serviceName, aclEnabled, true, Optional.of(tickInterval), servicePortMappings);
+    }
+
+    /**
+     * Add a service port mapping from serviceName to host:port.
+     * @return The updated settings.
+     */
+    public Settings withServicePortMapping(String serviceName, String hostAndPort) {
```

Comment:
hostAndPort looks it can leave room to error if you don't use `:` between the `host` and the `port`. I bet you considered two inputs, host and port. Why did you prefer `host` and `port` together?

Reply:
Correct. I might be better indeed. 

The reason I chose `host:port` is because we pass it as such in other places as well. 

For instance, if configuring it directly in docker-compose file, 
we need to pass: `-Dkalix.dev-mode.service-port-mappings.my-service-name=host.docker.internal:port`. When passing in one `-D` is convenient.

When configuring it in the maven plugin, we can pass just the port, and then we map it automatically to `localhost:port` or we can pass `somehost:port`. 

But here it might be better indeed to split it since it's always configurable in code.

Reply:
@franciscolopezsancho, done in e5200c0